src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -129,6 +130,21 @@ public void clear() {
         internalList.clear();
     }
 
+    /**
+     * Sorts the persons in list.
+     */
+    public void sort() {
+        internalList.sort(new Comparator<Person>() {
+            @Override
+            public int compare(Person p1, Person p2) {
```

Comment:
good, another way to improve the code quality further is to use lambda expressions.